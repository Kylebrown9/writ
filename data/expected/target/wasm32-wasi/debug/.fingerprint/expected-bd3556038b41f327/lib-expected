25096cce69c165cb