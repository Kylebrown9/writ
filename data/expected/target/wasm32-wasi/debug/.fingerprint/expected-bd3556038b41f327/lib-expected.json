{"rustc":6064785002308032717,"features":"[]","target":9778220589164093974,"profile":9251013656241001069,"path":17523903030608720598,"deps":[[18048171741498699465,"wit_bindgen_rust",false,3507220557789513140]],"local":[{"CheckDepInfo":{"dep_info":"wasm32-wasi/debug/.fingerprint/expected-bd3556038b41f327/dep-lib-expected"}}],"rustflags":[],"metadata":7797948686568424061,"config":2202906307356721367,"compile_kind":16489420942494921214}